<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssueTrackerClass</class>
 <widget class="QMainWindow" name="IssueTrackerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1623</width>
    <height>934</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IssueTracker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="issues_list_widget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>941</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_issue_line_edit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>740</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_issue_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>740</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Issue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_issue_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>810</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remove Issue</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="remove_issue_line_edit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>810</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resolve_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolve</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1623</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IssueTracker.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>add_issue_button</sender>
   <signal>clicked()</signal>
   <receiver>IssueTrackerClass</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>799</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>838</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remove_issue_button</sender>
   <signal>clicked()</signal>
   <receiver>IssueTrackerClass</receiver>
   <slot>remove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>861</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>676</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resolve_button</sender>
   <signal>clicked()</signal>
   <receiver>IssueTrackerClass</receiver>
   <slot>resolve()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>771</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>1447</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>issues_list_widget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>IssueTrackerClass</receiver>
   <slot>enable_resolve()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>713</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add()</slot>
  <slot>remove()</slot>
  <slot>resolve()</slot>
  <slot>enable_resolve()</slot>
 </slots>
</ui>
